changes/colw_include-vscode-config

```diff
@@ -0,0 +1 @@
+[Added] [#2807](https://github.com/cosmos/lunie/pull/2807) Include visual studio code config @colw
```

Comment:
For future reference: I would have used the "repository and tools" type

Reply:
Yep, good point. I got this right in the other repo just before.